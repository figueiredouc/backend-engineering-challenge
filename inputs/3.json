[
    {"timestamp": "2022-01-17 12:05:00.102000","translation_id": "1","source_language": "en","target_language": "fr","client_name": "client1","event_name": "translation_delivered","nr_words": 50, "duration": 305},
    {"timestamp": "2022-01-17 12:10:00.220000","translation_id": "2","source_language": "es","target_language": "de","client_name": "client2","event_name": "translation_delivered","nr_words": 30, "duration": 25},
    {"timestamp": "2022-01-17 12:15:00.300000","translation_id": "3","source_language": "fr","target_language": "en","client_name": "client3","event_name": "translation_delivered","nr_words": 40, "duration": 20}
]