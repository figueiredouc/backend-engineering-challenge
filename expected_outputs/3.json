[{"date": "2022-01-17 12:05:00", "average_delivery_time": 0},
 {"date": "2022-01-17 12:06:00", "average_delivery_time": 305.0},
 {"date": "2022-01-17 12:07:00", "average_delivery_time": 305.0},
 {"date": "2022-01-17 12:08:00", "average_delivery_time": 305.0},
 {"date": "2022-01-17 12:09:00", "average_delivery_time": 305.0},
 {"date": "2022-01-17 12:10:00", "average_delivery_time": 305.0},
 {"date": "2022-01-17 12:11:00", "average_delivery_time": 165.0},
 {"date": "2022-01-17 12:12:00", "average_delivery_time": 165.0},
 {"date": "2022-01-17 12:13:00", "average_delivery_time": 165.0},
 {"date": "2022-01-17 12:14:00", "average_delivery_time": 165.0},
 {"date": "2022-01-17 12:15:00", "average_delivery_time": 165.0},
 {"date": "2022-01-17 12:16:00", "average_delivery_time": 22.5}]